src/context/Theme.tsx

```diff
@@ -126,6 +126,13 @@ export const outlineSx: SxProps<Theme> = {
     border: `1px solid ${slate[200]}`,
 };
 
+// TODO (theme) Figure out how to make these composable
```

Comment:
Could you clarify this comment for me? I am not sure I understand the task it is identifying.